src/schema/sorts/partner_show_sorts.ts

```diff
@@ -74,6 +74,10 @@ const PartnerShowSorts = {
       START_AT_DESC: {
         value: "-start_at",
       },
+      PARTNER_ASC: {
+        // TODO: Change to proper order once Gravity supports this type of sort
+        value: "name",
```

Comment:
This is temporary of course, but for now should this just fall back to whatever our 2nd choice is for sorting (or whatever we're using right now)? 

If so, we might want `end_at` instead of `name`. I'm not positive so maybe we can get product to weigh in. 

Otherwise - awesome!

Reply:
oh didn't know whats the default sort, let me update.